### ISSUE TITLE
Tool template has a hard-coded icon color

### ISSUE THREAD

**Issue by blackfyre** [laravel/nova-issues#861]
- Laravel Version: 5.7.9
- Nova Version: 1.1.4
- PHP Version: 7.1.14

### Description:
The generated tool template has a hard-coded icon color which results in ignoring the theme settings.

### Steps To Reproduce:
Generate a tool and in the `nova-components/MyShinyTool/resources/views/navigation.blade.php` file there's the hard-coded fill on the icon's path: `fill="#B3C1D1"`

This should be changed to `fill="var(--sidebar-icon)"`

**Comment by alexbowers** [comment]
@blackfyre Can you confirm the version of Nova you are currently using?

**Comment by alexbowers** [comment]
I've confirmed this against the latest `1.1.4`

**Comment by blackfyre** [comment]
@alexbowers Sorry, was on a bank holiday and there was no internet...

**Comment by blackfyre** [comment]
Can confirm again, against `1.1.7`.

The stub is:
```html
<router-link tag="h3" :to="{name: '{{ component }}'}" class="cursor-pointer flex items-center font-normal dim text-white mb-6 text-base no-underline">
    <svg class="sidebar-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#B3C1D1" d="M3 1h4c1.1045695 0 2 .8954305 2 2v4c0 1.1045695-.8954305 2-2 2H3c-1.1045695 0-2-.8954305-2-2V3c0-1.1045695.8954305-2 2-2zm0 2v4h4V3H3zm10-2h4c1.1045695 0 2 .8954305 2 2v4c0 1.1045695-.8954305 2-2 2h-4c-1.1045695 0-2-.8954305-2-2V3c0-1.1045695.8954305-2 2-2zm0 2v4h4V3h-4zM3 11h4c1.1045695 0 2 .8954305 2 2v4c0 1.1045695-.8954305 2-2 2H3c-1.1045695 0-2-.8954305-2-2v-4c0-1.1045695.8954305-2 2-2zm0 2v4h4v-4H3zm10-2h4c1.1045695 0 2 .8954305 2 2v4c0 1.1045695-.8954305 2-2 2h-4c-1.1045695 0-2-.8954305-2-2v-4c0-1.1045695.8954305-2 2-2zm0 2v4h4v-4h-4z"/></svg>
    <span class="sidebar-label">
        {{ title }}
    </span>
</router-link>
```

instead of: 
```html
<router-link tag="h3" :to="{name: '{{ component }}'}" class="cursor-pointer flex items-center font-normal dim text-white mb-6 text-base no-underline">
    <svg class="sidebar-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="var(--sidebar-icon)" d="M3 1h4c1.1045695 0 2 .8954305 2 2v4c0 1.1045695-.8954305 2-2 2H3c-1.1045695 0-2-.8954305-2-2V3c0-1.1045695.8954305-2 2-2zm0 2v4h4V3H3zm10-2h4c1.1045695 0 2 .8954305 2 2v4c0 1.1045695-.8954305 2-2 2h-4c-1.1045695 0-2-.8954305-2-2V3c0-1.1045695.8954305-2 2-2zm0 2v4h4V3h-4zM3 11h4c1.1045695 0 2 .8954305 2 2v4c0 1.1045695-.8954305 2-2 2H3c-1.1045695 0-2-.8954305-2-2v-4c0-1.1045695.8954305-2 2-2zm0 2v4h4v-4H3zm10-2h4c1.1045695 0 2 .8954305 2 2v4c0 1.1045695-.8954305 2-2 2h-4c-1.1045695 0-2-.8954305-2-2v-4c0-1.1045695.8954305-2 2-2zm0 2v4h4v-4h-4z"/></svg>
    <span class="sidebar-label">
        {{ title }}
    </span>
</router-link>
```

**Comment by crnkovic** [comment]
This is fixed in `1.1.9`.

**Comment by blackfyre** [comment]
@crnkovic Can confirm it, closing the ticket.
